src/components/Authentication/userSignIn.js

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+import {Box, Button, InputAdornment, Link, Typography} from "@mui/material";
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import MailOutlineIcon from '@mui/icons-material/MailOutline';
+import Card from '@mui/material/Card';
+
+
+
+export default function UserSignIn() {
+
+
+    return (
+        <Card variant='outlined' style={{
+            width: '372px',
+            height: '785px',
+            justifyContent: 'center',
+            backgroundColor: '#E9E3D3',
+            margin: 'auto'
+        }}>
+            <p align='center' style={{
```

Comment:
Instead of using the style props, we should use a css file. This will keep the code cleaner.

Reply:
yes, I created an issue for the css file. As per my issue description, I need. to figure out how to override MUI css first. This is a quick fix so we can have something to work out (ie connected FE/BE together)